modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ViewObjectEntriesDisplayContext.java

```diff
@@ -148,7 +148,8 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems()
 			new FDSActionDropdownItem(
 				LanguageUtil.get(
 					_objectRequestHelper.getRequest(),
-					"are-you-sure-you-want-to-delete-this-entry"),
+					"it-may-affect-many-records-are-you-sure-you-want-to-" +
+						"delete-this-entry"),
```

Comment:
@diogomelo09 Could you confirm the expected behavior in the ticket? 

Because it says: 

> Actual Result:
> No information message is displayed. 

but we already have the confirmation message `Are you sure you want to delete this entry?` when trying to delete any entry. 

The ticket description is mentioning the use case of deleting an **Entry cascade**, so I believe your fix should be more specific for this use case.

Thanks!

Reply:
@diogomelo09 I talked to the QAs and they updated the ticket description.

>  Actual Result:
> The error message is not specific.

Could you update the Poshi test with the new message [here](https://github.com/liferay/liferay-portal/blob/045dfcd7cf0f5630008e30b11e4694570adea96a/modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro#L321)?

Thanks!

Reply:
Done!! :grin: :tada: